src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -509,7 +509,7 @@ internal class Tokenizer
 
         // Unicode replacement char used to represent an unknown, unrecognized or unrepresentable character in a
         // Unicode escape sequence.
-        private static readonly string s_unknownUnicodeChar = ((char)0xffdd).ToString();
+        private static readonly string s_unknownUnicodeChar = ((char)0xfffd).ToString();
```

Comment:
Hehe, I wanted to correct the **original typo** for replacement/unknown character.  The correct value for that is `0xfffd`.  Next up, I'm looking at switching to what C# uses - invalidChar which is Char.MaxValue (0xffff).